public/locales/en/translations.json

```
@@ -0,0 +1,51 @@
+	"roadmap.soft_fork.2": "The initial forking off of Nix and NixPkgs will be performed in this phase and new contributors will be given roles for handling package maintenance and pulling changes from upstream. We intend to move on from this phase once enough contributors have joined for the project to be capable of transitioning into a hard fork.",
+	"roadmap.hard_fork.title": "Hard Fork",
+	"roadmap.hard_fork.1": "Like the Soft Fork phase, management structure will still be ad-hoc, but Committees, Special Interest Groups, and Working Groups may start to be formed. Due to there being necessary bootstrapping steps for each, elections will only begin once the Alignment phase has been reached.",
+	"roadmap.hard_fork.2": "At this point we will have enough contributors to safely maintain NixPkgs (or a reasonable subset) and Nix ourselves. We will begin allowing divergence from the upstream repository and changes to the repository structure to improve development can be introduced.",
+	"roadmap.hard_fork.3": "An initial release of the packages repository will be created and migration from `nix` to `aux` naming will begin to reduce confusion and errors now that the two projects have diverged.",
+	"roadmap.organization.title": "Organization",
+	"roadmap.organization.1": "The packages repository will have sets extracted to allow for Special Interest Groups to more easily manage their lifecycles. For example, `pythonPackages` will be moved to a separate repository that will be the reponsibility of a Python SIG if there is interest.",
+	"roadmap.organization.2": "Flakes will be standardized with its current implementation as a v0. While not ideal, the feature is used far too widely to be changed or removed without breaking the ecosystem. Instead, this v0 implementation will be enabled and future work for Flakes that addresses its shortcomings may be handled by a Flakes SIG.",
+	"roadmap.organization.3": "The `aux` CLI will be modified to provide more ergonomic management of packages and systems. Additional subcommands such as `aux system switch` and `aux system build` will be added to make onboarding and ongoing maintenance easier.",
+	"roadmap.infa.title": "Infastructure",
+	"roadmap.infa.1": "Now that the project has significantly diverged from upstream, we will need to provide our own Continuous Integration and Binary Cache services. Existing governance structures will be used to manage the adoption of these technologies.",
+	"roadmap.infa.2": "Documentation will be included as a part of the Continuous Integration process and will have Continuous Deployment configured to keep it up to date, making it easier for contributors to create educational materials.",
+	"roadmap.infa.3": "Search infrastructure for packages and options will also be created.",
```

Ditto here with `infra`
```suggestion
	"roadmap.infra.title": "Infrastructure",
	"roadmap.infra.1": "Now that the project has significantly diverged from upstream, we will need to provide our own Continuous Integration and Binary Cache services. Existing governance structures will be used to manage the adoption of these technologies.",
	"roadmap.infra.2": "Documentation will be included as a part of the Continuous Integration process and will have Continuous Deployment configured to keep it up to date, making it easier for contributors to create educational materials.",
	"roadmap.infra.3": "Search infrastructure for packages and options will also be created.",
```